tests/e2e-pw/specs/merchant/merchant-admin-account-balance.spec.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { useMerchant } from '../../utils/helpers';
+
+test.describe(
+	'Merchant account balance overview for single deposit currency accounts',
+	() => {
+		// Use the merchant user for this test suite.
+		useMerchant();
+
+		test(
+			'View the total and available account balance for a single deposit currency (USD)',
+			{
+				tag: '@critical',
+			},
+			async ( { page } ) => {
+				await test.step(
+					'Navigate to the Payments Overview screen',
+					async () => {
+						await page.goto(
+							'/wp-admin/admin.php?page=wc-admin&path=/payments/overview'
+						);
+					}
+				);
+
+				await test.step(
+					'Observe the total account balance, ensuring it has a dollar value with the correct currency formatting',
+					async () => {
+						const totalBalanceValue = page.getByLabel(
+							'Total balance',
+							{
+								exact: true,
+							}
+						);
+
+						// Match the total balance value to the USD format $1*
+						// Intentionally not expecting a specific value to allow for different values in different environments.
+						await expect( totalBalanceValue ).toHaveText( /\$\d+/ );
+					}
+				);
+
+				await test.step(
+					'Observe the available account balance, ensuring it has a dollar value with the correct currency formatting',
+					async () => {
+						const availableFundsValue = page.getByLabel(
+							'Available funds',
+							{
+								exact: true,
+							}
+						);
+						await expect( availableFundsValue ).toHaveText(
+							/\$\d+/
+						);
+					}
+				);
+			}
+		);
+	}
+);
+
+test.describe(
+	'Merchant account balance overview for multiple deposit currency accounts',
+	{
+		tag: '@critical @todo',
```

Comment:
Also, I would include a [test annotation](https://playwright.dev/docs/test-annotations#annotate-tests) in every single place where we add the `@todo` tag, in order to link the placeholder with technical debt and improve traceability.

Ideally, and pending test (or suite of tests) to be implemented should have a GitHub issue that address it, and we could link that issue to the test by using these annotations.

Example:
```
 annotation: {
    type: 'issue',
    description: 'https://github.com/automattic/woocommerce-payments/issues/99999',
  },
```
Or even:
```
 annotation: {
    type: 'technical debt',
    description: 'https://github.com/automattic/woocommerce-payments/issues/99999',
  },
```

I think we can add anything we want to the `type` parameter so it is up to us.

Reply:
> Defining a pattern for what to name these will be useful. I'm not sure what to name these right now but can create a starting point.

+1

> technical debt may be less obvious, could be a link to an issue or a text description?

In case of technical debt, I would use it to point to the GH issue. I think it could be mostly useful if we want to quality why we want to link a test to an issue, i.e. if it's to provide extra context, or to indicate that it will be implemented in the future (technical debt), etc.

I believe in the power of simplicity in software development, maybe we can stick to 'issue' initially and decide later if we need a better classification system as we start annotating tests?

Reply:
Sounds good!

I've added an `issue` annotation and `description` for a text description of the blocker for the `@todo` test in 938b8f8.